commands/promote.js

```diff
@@ -50,6 +53,8 @@ function * run (context, heroku) {
     promotionMessage = `Promoting ${cli.color.addon(attachment.addon.name)} to ${cli.color.configVar('DATABASE_URL')} on ${cli.color.app(app)}`
   }
 
+  cli.warn('Detaching DATABASE. If you are using the release phase feature (https://devcenter.heroku.com/articles/release-phase), this can cause a failed release.')
```

Comment:
can it be a bit more generic to `pg:promote`? For example, the real issue here is that the release phase can fail for either detach or attach, or even both. Failing only detach is not that much critical if attach still works. OTOH, failing attach is pretty bad -- it can cause "without a database" situation, or if both detach attach failed, you could keep using the old database without knowing it (as the app would keep "working").
I think it's good to generalize the message and be something like "if you're using the release phase, pg:promote can cause the failure release. please double check the releases and make sure database is properly promoted".